alsa_settings/MTLP_RVP_SDW.sh

```diff
@@ -1,5 +1,9 @@
 set -e
 
+# override jack detection mode to headset
+# related linux pr: https://github.com/thesofproject/linux/pull/4969
+amixer -c sofsoundwire cset name='rt711 GE49 Selected Mode' 2 || true
```

Comment:
I just realized that this is a copy/paste of 90e0929368cff0 in `LNLM_SDW_AIOC.sh`

Can you please add a duplication warning at the top of the file? Something short and generic, no need to go into a detailed comparison about what is copied from `LNLM_SDW_AIOC` versus what has diverged. Something like: `# A lot of this file is duplicated in LNLM_SDW_AIOC.sh and possibly others.`

Reply:
> I just realized that this is a copy/paste of [90e0929](https://github.com/thesofproject/sof-test/commit/90e0929368cff0b29bfa92475bc4893717a5ebd1) in `LNLM_SDW_AIOC.sh`
> 
> Can you please add a duplication warning at the top of the file? Something short and generic, no need to go into a detailed comparison about what is copied from `LNLM_SDW_AIOC` versus what has diverged. Something like: `# A lot of this file is duplicated in LNLM_SDW_AIOC.sh and possibly others.`
> 
> Also add the same warning at the top of `LNLM_SDW_AIOC.sh`, which will avoid future commits like [90e0929](https://github.com/thesofproject/sof-test/commit/90e0929368cff0b29bfa92475bc4893717a5ebd1) missing the duplication and forgetting to update some files.

Done